contracts/LimitOrder.sol

```diff
@@ -435,13 +421,12 @@ contract LimitOrder is ILimitOrder, BaseLibEIP712, SignatureValidator, Reentranc
                 remainingAmount: _settlement.remainingAmount,
                 makerTokenFee: 0,
                 takerTokenFee: takerTokenFee,
-                takerTokenProfit: takerTokenProfit,
-                takerTokenProfitFee: takerTokenProfitFee,
-                takerTokenProfitBackToMaker: takerTokenProfitBackToMaker
+                relayerTakerTokenProfit: relayerTakerTokenProfit,
+                relayerProfitFee: relayerProfitFee
```

Comment:
Do you think it is worth to emphasize fee is in taker token like `relayerTakerTokenProfitFee`, which is also aligned with other variable naming.

Reply:
I thought about it but it may be a little bit lengthy. I'm ok with both. cc @NIC619

Reply:
Sounds good to me!